<launch>
  <node pkg="topic_tools" type="throttle" name="throttle" args="messages /camera/depth_registered/points 1.0 cloud_in">
  </node>

  <arg name="start_manager" default="false"/>
  <arg name="nodelet_manager" default="camera_nodelet_manager"/>
  <node if="$(arg start_manager)" pkg="nodelet" type="nodelet" name="$(arg nodelet_manager)"  args="manager" output="screen"/>
  <node pkg="nodelet" type="nodelet" name="segmentation" args="load cob_3d_segmentation/SegmentationAllInOneNodelet $(arg nodelet_manager)" output="screen">
    <remap from="cloud_in" to="registration/pc_aligned"/>
    <remap from="shape_array" to="/plane_extraction/shape_array"/>
  </node>
</launch>